templates/js/non-sso-tab/src/views/hello.html

```diff
@@ -5,7 +5,7 @@
     <link rel="icon" href="/favicon.ico" />
     <link rel="stylesheet" type="text/css" href="/static/styles/custom.css" />
     <script
-      src="https://res.cdn.office.net/teams-js/2.17.0/js/MicrosoftTeams.min.js"
+      src="https://res.cdn.office.net/teams-js/2.19.0/js/MicrosoftTeams.min.js"
```

Comment:
What about using the latest version?

Reply:
Ok, I'll try to update it to the latest version: 2.22.0 and update after verifying that it works.